loaders/src/glTF/2.0/babylon.glTFLoader.ts

```diff
@@ -600,63 +600,67 @@ module BABYLON.GLTF2 {
             if (primitive.targets !== undefined) {
                 morphTargetManager = new BABYLON.MorphTargetManager();

```

Comment:
I'm a bit confused. Is it intended that you are creating multiple MorphTargetManager objects per mesh?